src/App.js

```diff
@@ -3,15 +3,57 @@ import Header from './Header.js';
 import Footer from './Footer.js';
 import Main from './Main.js';
 import data from './data.json'
+import Modal from 'react-bootstrap/Modal'
 
 import './App.css';
 
 class App extends React.Component{
+  constructor(props){
+    super(props);
+    this.state={
+      show: false,
+      animal:{title: 'some beast'},
+      // title: this.props.animal.title
+    }
+  }
+
+  handleShow = (animal) => {
+    this.setState({
+      show: true,
+      animal
+    })
+  }
+  handleClose = () => {
+    this.setState ({
+      show:false
+    })
+  }
+  
   render(){
     return(
       <>
         <Header />
-        <Main data={data}/>
+
+        <Modal 
+          show={this.state.show} 
+          onHide={this.handleClose}
+          centered
+          size="xl"
+          >
+          <Modal.Header closeButton>
+            <Modal.Title>{this.state.animal.title}</Modal.Title>
+          </Modal.Header>
+          <Modal.Body>
+            <img 
+            src={this.state.animal.image_url}
+            className="w-100" 
+            alt={this.state.animal.title}
+            />
+            {this.state.animal.description}
+          </Modal.Body>
+        </Modal>
```

Comment:
Good job so far on this! 👍 

- You asked what pieces of code go where so I want to lend a hand :)
- The lab wants us to create a SelectedBeast component (step 1)
- We should import that into this component (App)
- From there, this code is what we want to see in your newly created SelectedBeast component

Grab a TA when you can so they can lend a hand too 👍